app/src/main/java/org/mifos/mobile/data/PaymentHubApiManager.kt

```diff
@@ -0,0 +1,64 @@
+package org.mifos.mobile.data
```

Comment:
I see a new package being created `data`, what's the purpose. clarify

Reply:
I didn't find any relevant place for placing these data files of `PaymentHub`, therefore I decided to create a separate package for them to place all of them. Please let me know if its correct or not. And if not, then please specify where should I place them.

Reply:
add it inside `api` package